index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Portfolio Setup and Mobile first</title>
+  </head>
+  <body>
+    <header class="logo-menu">
+      <div class="logo" onclick="window.location.href='index.html'">
+        <a href="index.html">My Logo</a>
+      </div>
+      <nav class="fa fa-bars bars"></nav>
+    </header>
+    <main>
+      <div class="main-container">
+        <section class="titleText">
```

Comment:
- [ ] [REQUIRED] The whole container of the section should be a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info, please read carefully that info on how to implement the `section` tag because `section` tag used before the `<div class="main-container">` e,t,c 🙏🏼

Reply:
implemented